modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -41,6 +47,21 @@ $ oc patch networks.operator.openshift.io cluster --type=merge -p \
 ----
 +
 <1> Specify `External` to encrypt traffic to external hosts or specify `Full` to encrypt pod-to-pod traffic and, optionally, traffic to external hosts. By default, IPsec is disabled.
++
+.Example configuration that has IPsec enabled in `Full` mode and `encapsulation` set to `Always`
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p \
+  '{
+  "spec":{
+    "defaultNetwork":{
+      "ovnKubernetesConfig":{
+        "ipsecConfig":{
+          "mode":"Full"
```

Comment:
Missing a comma after ` "mode":"Full"`

Correct one is as below
```
%  oc patch networks.operator.openshift.io cluster --type=merge -p \
  '{
  "spec":{
    "defaultNetwork":{
      "ovnKubernetesConfig":{
        "ipsecConfig":{
          "mode":"Full",
          "full":{
            "encapsulation": "Always"
          }}}}}}'
network.operator.openshift.io/cluster patched
```

Reply:
:heavy_check_mark: